### PR TITLE
[device_info_platform_interface] null safety stable release

### PR DESCRIPTION
**Pull request by cyanglaz**
Move null safety to stable

### CODE REVIEW

File: packages/device_info/device_info_platform_interface/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
 * Make `baseOS`, `previewSdkInt`, and `securityPatch` nullable types.
 * Remove default values for non-nullable types.
-
-## 2.0.0-nullsafety.1
-
-* Bump Dart SDK to support null safety.
-
-## 2.0.0-nullsafety
-
 * Migrate to null safety.
```
Comment (stuartmorgan) [inline packages/device_info/device_info_platform_interface/CHANGELOG.md:5]: Nit: let's make this first since it's the key change in the 2.0.0 release.

**Review by stuartmorgan — APPROVED** [review]
LGTM with nit

File: packages/device_info/device_info_platform_interface/CHANGELOG.md (hunk 1 above)
Comment (cyanglaz) [reply packages/device_info/device_info_platform_interface/CHANGELOG.md]: done

**Comment by cyanglaz** [comment]
Do not publish, device_info_platform_interface's code caused https://github.com/flutter/flutter/issues/76194, need to fix it before moving it to stable

**Pull request merged**